client.cpp

```diff
@@ -423,7 +423,12 @@ void client::handle_event(short evtype)
         new_evtype = EV_READ;
     }
     if (evbuffer_get_length(m_write_buf) > 0) {
-        assert(finished() == false);
```

Comment:
@tomerr-lb  i`m not sure i understand why ? it means something is stuck as socket buffer is completely full and does not advance ..

Reply:
@sashas-lb when running with test timout, there is always a possibility that when the timeout expires, some of the requests are still waiting in the write buffer to be sent.